src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Assembly.java

```diff
@@ -44,4 +47,15 @@
     @Column(nullable = false)
     private String name;
 
+    @ApiModelProperty(position = 3, required = true)
+    @JsonProperty
+    @Column(nullable = true)
+    private String patch;
+
+    @ApiModelProperty(position = 4, required = true)
+    @NotNull
+    @JsonProperty
+    @ElementCollection
+    private List<String> accessions = new ArrayList<String>();
+
```

Comment:
sorry why do we need list here ?

Reply:
There are cases in which an assembly version have different accessions from different accession issuers. For example, the laster human reference genome GRCh38.p12, the GenBank assembly accession is GCA_000001405.27 and the RefSeq assembly accession is GCF_000001405.38.

Reply:
https://www.ncbi.nlm.nih.gov/assembly/GCF_000001405.38/

Reply:
Oh thanks i understand now.